pkg/porter/generateManifest.go

```diff
@@ -111,15 +113,83 @@ func (p *Porter) generateInternalManifest(ctx context.Context, opts BuildOptions
 			}
 		}
 		return e.SetValue(path+"digest", digest.String())
-
 	})
 	if err != nil {
 		return err
 	}
 
+	if p.IsFeatureEnabled(experimental.FlagDependenciesV2) {
```

Comment:
Same as below (?) - I would love to see this moved into [here](https://github.com/getporter/porter/blob/main/pkg/manifest/manifest.go#L120) - [we can use this](https://github.com/getporter/porter/blob/main/pkg/manifest/manifest.go#L174) too :)